infer.py

```diff
@@ -23,10 +23,10 @@ def _build(self):
 
     def load(self):
         path = os.path.join(self.checkpoint_directory, "best.ckpt")
-        assert os.path.exists(path)
-        state = torch.load(path)
-        self.model.load_state_dict(state["model"])
-        self.model.eval()
+        if os.path.exists(path):
```

Comment:
could we add a warning here that gets triggered if the path does not exist? I'm afraid of silent fails...